R/learner_randomPlantedForest_regr_rpf.R

```diff
@@ -0,0 +1,90 @@
+#' @title Regression Random Planted Forest Learner
+#' @author jemus42
+#' @name mlr_learners_regr.rpf
+#'
+#' @description
+#' Random Planted Forest: A directly interpretable tree ensemble.
+#'
+#' Calls [randomPlantedForest::rpf()] from 'randomPlantedForest'.
+#'
+#' @inheritSection mlr_learners_classif.rpf Custom mlr3 parameters
+#' @templateVar id regr.rpf
+#' @template learner
+#' @inheritSection mlr_learners_classif.rpf Installation
+#'
+#' @references
+#' `r format_bib("hiabu_2023")`
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+LearnerRegrRandomPlantedForest = R6Class("LearnerRegrRandomPlantedForest",
+  inherit = LearnerRegr,
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      param_set = ps(
+        max_interaction       = p_int(lower = 0, upper = Inf, default = 1, tags = "train"),
+        max_interaction_ratio = p_dbl(lower = 0, upper = 1, tags = "train"),
+        max_interaction_limit = p_int(lower = 1, upper = Inf, tags = c("required", "train"), special_vals = list(Inf)),
+        ntrees                = p_int(lower = 1, upper = Inf, default = 50, tags = "train"),
+        splits                = p_int(lower = 1, upper = Inf, default = 30, tags = "train"),
+        split_try             = p_int(lower = 1, upper = Inf, default = 10, tags = "train"),
+        t_try                 = p_dbl(lower = 0, upper = 1, default = 0.4, tags = "train"),
+        deterministic         = p_lgl(default = FALSE, tags = "train"),
+        nthreads              = p_int(lower = 1, upper = Inf, default = 1, tags = c("train", "threads")),
```

Comment:
one more question: is this parameter really only active during train? (You fixed this but for ranger :D)

Reply:
Yep - when I rewrote that part of the upstream learner to use an integer `nthreads` rather than a boolean `parallel` parameter I noticed it's only referenced in C++ functions with names like `fit()` and `get_parameters()` 😅